posixlib/src/main/resources/scala-native/types.h

```diff
@@ -1,6 +1,8 @@
 #ifndef __TYPES_H
 #define __TYPES_H
 
+// types used in pwd.c and sys/stat.c
```

Comment:
OK, why?  I know the answer but that is surely the question that a new reader will ask.

Reply:
I put that there so the reader could know it is used only.